SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -15155,7 +15159,7 @@ void CEulerSolver::LoadRestart(CGeometry **geometry, CSolver ***solver, CConfig
       }
       
     }
-    iPoint_Global++;
+    iPoint_Global_Local++;
   }
```

Comment:
This indeed fixed the problem, thanks! Should we apply these changes to the other Solvers as well (eg. turbulence, wave ..) ?